sw/device/lib/dif/dif_entropy.h

```diff
@@ -98,35 +98,6 @@ typedef enum dif_entropy_test {
   kDifEntropyTestNumVariants,
 } dif_entropy_test_t;
 
-/**
- * A mode of operation for the entropy source.
- */
-typedef enum dif_entropy_mode {
-  /**
-   * Indicates that the source is disabled.
-   */
-  kDifEntropyModeDisabled = 0,
-
-  /**
-   * The physical true random number generator mode.
-   *
-   * This mode uses a physical random noise generator for operation, and is
-   * truly random. This noise generator is compatible with SP 800-90B.
-   */
-  kDifEntropyModePtrng = 1,
-
-  /**
-   * The Linear Feedback Shift Register (LFSR) mode.
-   *
-   * This mode is digital, and as such is only pseudo-random and intended
-   * for test purposes only.
-   *
-   * In this mode, the `dif_entropy_config.lfsr_seed` value is used to
-   * initialize the internal state of the LFSR.
-   */
-  kDifEntropyModeLfsr = 2,
-} dif_entropy_mode_t;
-
 /**
```

Comment:
In my hack version i keep this struct (minimize changes), but getting rid of it is good also.